lldb/source/DataFormatters/TypeFormat.cpp

```diff
@@ -96,16 +96,19 @@ bool TypeFormatImpl_Format::FormatObject(ValueObject *valobj,
 
         ExecutionContextScope *exe_scope =
             exe_ctx.GetBestExecutionContextScope();
-        std::optional<uint64_t> size = compiler_type.GetByteSize(exe_scope);
-        if (!size)
+        auto size_or_err = compiler_type.GetByteSize(exe_scope);
+        if (!size_or_err) {
+          LLDB_LOG_ERRORV(GetLog(LLDBLog::Types), size_or_err.takeError(),
+                          "{0}");
```

Comment:
Do we want to be more descriptive in the log? Just so someone reading the log can map back to where in the source this came from? Same applies to a couple of other places where you added logging